src/History/HistoryItem.jsx

```diff
@@ -1,18 +1,30 @@
 import PropTypes from "prop-types";
 
+import sample from "../../sample.json";
 import DeleteButton from "../shared/DeleteButton";
 import SearchKeyword from "./SearchKeyword";
 import SearchUrl from "./SearchUrl";
+import { TotalKeywordButton } from "./TotalKeywordButton";
 
 export default function HistoryItem({ keywords, onDragStart, onDrop }) {
+  const data = sample;
+  let siteUrl;
+  let siteName;
+  for (let i = 0; i < data.length; i++) {
+    siteUrl = data[i].siteTitle;
+  }
+  for (let i = 0; i < data.length; i++) {
+    siteName = data[i].url;
+  }
```

Comment:
P2 : 해당 부분 다음과 동일한 결과 같아요. 이렇게 개선해보는 것은 어떻까요?

```suggestion
  const siteUrl = data[data.length-1].siteTitle;
  const siteName = data[data.length-1].url;
```

Reply:
넵 의견 주셔서 감사합니다:)